<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAffine</class>
 <widget class="QDialog" name="DialogAffine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>202</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Affine Transformation</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_2"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_5"/>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="lineEdit_4"/>
         </item>
         <item row="0" column="0">
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="lineEdit_3"/>
         </item>
         <item row="2" column="2">
          <widget class="QLineEdit" name="lineEdit_9"/>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="lineEdit_6"/>
         </item>
         <item row="2" column="0">
          <widget class="QLineEdit" name="lineEdit_7"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_8"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogAffine</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogAffine</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
